### ISSUE TITLE
How to construct training set for SFT training

### ISSUE THREAD

**Issue by Se-Hun** [swt-user/DMPO#2]
First of all, thank you for sharing such a great approach!

I have a question.
The training dataset(ex. `webshop_sft.json`) for SFT already has **reward values** labeled. How were these values extracted?

**Comment by swt-user** [comment]
Thanks for the interest! The SFT training dataset construction details can be found in [ETO](https://arxiv.org/pdf/2403.02502#page=11.08)
